py-polars/polars/expr/string.py

```diff
@@ -2030,34 +2029,33 @@ def explode(self) -> Expr:
         """
         return wrap_expr(self._pyexpr.str_explode())
 
-    def parse_int(self, radix: int | None = None, *, strict: bool = True) -> Expr:
+    def to_integer(self, base: int = 10, *, strict: bool = True) -> Expr:
```

Comment:
Could we make `base` keyword-only for now? That way, it will be easier to add a `dtype` argument later if we want to.

```suggestion
    def to_integer(self, *, base: int = 10, strict: bool = True) -> Expr:
```